src/reducers/dataset.test.ts

```diff
@@ -11,8 +11,7 @@ describe('reducers/dataset', () => {
       expect(datasetReducer(DEFAULT_DATASET, {
         type: DATASET_REQUEST,
         payload: {
-          name: 'cars',
-          url: 'http://cars.com'
```

Comment:
Why?

Reply:
I was because that the url was not specified as a property in payload.